augur/refine.py

```diff
@@ -228,7 +228,12 @@ def run(args):
                         clock_filter_iqd=args.clock_filter_iqd,
                         covariance=args.covariance, resolve_polytomies=(not args.keep_polytomies))
         except BaseException as err:
-            raise AugurError(f"Was unable to refine time trees:\n\n{err}")
+            if type(err).__name__ == "TreeTimeError":
+                raise AugurError(f"Was unable to refine time trees:\n\n{err}")
+            else:
+                import traceback 
+                traceback.format_exc()
+                raise err
```

Comment:
This would read better to me:

```suggestion
        except TreeTimeError as err:  # add import at the top of file
            raise AugurError(f"Was unable to refine time trees:\n\n{err}")
        except BaseException as err:
            import traceback 
            traceback.format_exc()
            raise err
```